components/02-molecules/alert/_yds-alert.scss

```diff
@@ -24,7 +25,7 @@ $alert-themes: map.deep-get(sass-tokens.$tokens, color, alert);
   }
 
   &[data-alert-state='dismissed'] {
-    display: none;
+    @include tokens.animate-fade-to-hidden($alert-fade-speed);
```

Comment:
Here, we can use the animate-hidden mixin with the speed variable we set at the top of this file.

```
@include tokens.animate-hidden($alert-fade-speed);
```

Then, we need to set the new "visual" attributes. I'm thinking something like:

```
    opacity: 0;
    max-height: 0;
    overflow: hidden;
```